docs/DEPLOY.md

```diff
@@ -0,0 +1,53 @@
+
+## Deploy
+Deployment of team web on team10-20.studenti.fiit.stuba.sk.
+
+#### Steps
+1. Build the app locally:
+
+	```shell script
+	npm run build
+	```
+	This creates /dist folder which content is uploaded on server. 
+
+1. Connect to server using ssh:
+
+	```shell script
+	ssh -i team10.key ubuntu@team10-20.studenti.fiit.stuba.sk
+	```
+
+1. Setup sftp server in FileZilla via File -> Site Manager -> New site:
```

Comment:
Also, should we document the exact program used to deploy? I would just refer to it as `SFTP Client` instead of `FileZilla` - there are lot of clients out there